docs/static/breaking-changes.asciidoc

```diff
@@ -29,7 +30,47 @@ and have the persistent queue enabled, we strongly recommend that you drain or
 delete the persistent queue before you upgrade. See <<upgrading-logstash-pqs>>
 for information and instructions.
 
+
+[[breaking-7.0]]
+=== Breaking changes in 7.0
+
+Here are the breaking changes for 7.0. 
+
+[float]
+==== Changes in Logstash Core
+
+These changes can impact any instance of Logstash and are plugin agnostic, but only if you are using the features that are impacted.
+
+
+[float]
+==== Changes in Logstash Plugins
+
+Here are the changes to plugins.
+
 [float]
+===== Beats input
+
+Beats fields have been renamed to map to Elastic Common Schema (ECS).
+
+Starting with 7.0, the fields exported by {beats} conform to the
```

Comment:
:thinking:

These breaking changes aren't in the Beats Input itself, but in the payloads that Beats sends. If a user upgrades Logstash before upgrading Beats, the payloads will continue to have the pre-ECS schema, and if they upgrade their Beats before upgrading Logstash they'll get payloads with the ECS schema in advance of any Logstash upgrade.

I would rather add this as a call-out than in bullet-points. See my proposed diff above for a `WARNING` call-out.